walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,100 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Iterator;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
 
+
+/*  Find time slot(s) that all the required attendees are availble
+    Include optional attendees if at least one time slot can be found
+    Otherwise return the time slots that fit just the mandatory attendees.
+*/
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
1) Generally best practice to document public methods, classes, interfaces, etc.
2) What does the method "query" do if no meeting slots are found? Should we document this?
3) Another case: what happens if required attendees have no available time slots, but optional attendees do?